Maskbook/Scene/App/FIleService/View/FileServiceView.swift

```diff
@@ -164,6 +169,20 @@ final class FileServiceOnboardViewModel: ObservableObject {
             .union(items)
             .map { $0 }
     }
+    
+    func addFromLocalfiles() {
+        coordinator.present(scene: .fileServiceLocalFileSource(selectFileHandler: selectFileHandler), transition: .panModel(animated: true))
```

Comment:
Should use `.panModel()` instead of `.panModel(animated: true)`, `enum` now supports default parameter.